prompt/prompt-executor/prompt-executor-clients/prompt-executor-ollama-client/src/commonMain/kotlin/ai/koog/prompt/executor/ollama/client/OllamaClient.kt

```diff
@@ -25,21 +25,23 @@ import kotlinx.coroutines.flow.flow
 import kotlinx.serialization.json.Json
 
 /**
- * Client for interacting with the Ollama API.
+ * Client for interacting with the Ollama API with comprehensive model support.
  *
  * @property baseUrl The base URL of the Ollama API server.
  * @property baseClient The HTTP client used for making requests.
+ * @property timeoutConfig Timeout configuration for HTTP requests.
+ * @property enableDynamicModels Whether to enable dynamic model resolution.
  */
 public class OllamaClient(
     private val baseUrl: String = "http://localhost:11434",
     baseClient: HttpClient = HttpClient(engineFactoryProvider()),
-    timeoutConfig: ConnectionTimeoutConfig = ConnectionTimeoutConfig()
+    timeoutConfig: ConnectionTimeoutConfig = ConnectionTimeoutConfig(),
+    private val enableDynamicModels: Boolean = true
```

Comment:
I propose that we remove that and use the model names advertised by Ollama directly without any `"ollama-"` or `"dynamic-"` prefixes.